src/material/expansion/expansion-panel-header.ts

```diff
@@ -178,6 +154,20 @@ export class MatExpansionPanelHeader implements OnDestroy, FocusableOption {
     return !this.panel.hideToggle && !this.panel.disabled;
   }
 
+  /**
+   * Gets the current height of the header. Null if no custom height has been
+   * specified, and if the default height from the stylesheet should be used.
+   */
+  _getHeaderHeight(): string|null {
+    const isExpanded = this._isExpanded();
+    if (isExpanded && this.expandedHeight) {
+      return `${this.expandedHeight}px`;
```

Comment:
`expandedHeight` and `collapsedHeight` are inputs that come in like `56px` - you should remove the `px` suffix here